.editorconfig

```
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+max_line_length = 120
+tab_width = 4
+ij_continuation_indent_size = 8
```

One last little suggestion just for QoL not needed just a good thing to remove any in between white spaces. Skript suffers a lot from this in older files I've learned recently.

```suggestion
max_line_length = 120
tab_width = 4
trim_trailing_whitespace = true
ij_continuation_indent_size = 8
```